README.md

```diff
@@ -5,9 +5,10 @@ Small repo done in cab ride
 A simple repo used to track progress of codechef monthly contests from JAN15-JUNE17 (Can be increased or decresed by modifying code)
 
 ## What to do?
-* Install virtual env
+* Install virtual env `sudo apt-get install virtualenv`
 * Create an environment using virtual env `virtualenv envName -p python3`
-* Activate the environment and install the requirements `source bin/activate; pip3 install -r req.txt`
+* Activate the environment and install the requirements `source envName/bin/activate; pip3 install -r req.txt`
```

Comment:
Better to make it, cd envName and then source bin/activate :P The location of probelms is an issue here.